src/wpt/lib/get_result.parser.js

```diff
@@ -0,0 +1,71 @@
+import { z } from 'https://esm.sh/zod@v3.20.2';
+
+const asset_type = z.object({
+	color: z.tuple([z.number(), z.number(), z.number()]),
+	bytes: z.number(),
+});
+
+/** @typedef {import("https://esm.sh/zod@v3.20.2").infer<request>} Request */
```

Comment:
If we’re using the `z` import, I think we can simplify this:

```suggestion
/** @typedef {z.infer<request>} Request */
```

Reply:
ah nice